dashboard/app/controllers/activities_controller.rb

```diff
@@ -155,6 +155,7 @@ def track_progress_for_user
     attributes = {
       user: current_user,
       level: @level,
+      script: @script_level&.script,
```

Comment:
I'm adding `script_id` to `activities` because there is nothing preventing an assessment from appearing in 2 scripts, and if it does then we want to be able to observe the student's activity pattern limited to a particular script or script level.